Content.Server/Ghost/Roles/Components/GhostRoleComponent.cs

```diff
@@ -63,6 +65,18 @@ public string RoleRules
             }
         }
 
+        [ViewVariables(VVAccess.ReadWrite)]
+        [Access(typeof(GhostRoleSystem), Other = AccessPermissions.ReadWriteExecute)] // FIXME Friends
+        public bool RoleAntagonist
+        {
+            get => _roleAntagonist;
+            set
+            {
+                _roleAntagonist = value;
+                EntitySystem.Get<GhostRoleSystem>().UpdateAllEui();
+            }
+        }
```

Comment:
Why does this update EUI if it's not UI facing?

Also EntitySystem.Get<T> is obsolete.